FlightReservation/components/flightsComponents/airplaneIcon.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {StyleSheet, View, Text, Image} from 'react-native';
+import colors from '../../src/colors';
+import airplane from '../../img/flight.png';
+
+function AirplaneIcon(){
+    return(
+        <View>
+          <Image
+            style={[styles.iconCheck, {marginTop: 20}]}
+            source={airplane}
+          />
+        </View>
+    )
+}
+const styles = StyleSheet.create({
+    iconCheck: {
+      tintColor: colors.bluePrimary,
+      height: 20,
+      width: 25,
+    },
+  });
+  
+  export default AirplaneIcon;
```

Comment:
indentation here has 4 spaces, but you are using 2 spaces among all the project. We need to be consistent on this. I recommend you to use 2 spaces instead of 4.

Reply:
done, we just applied prettier